From c36dc656049c397541650f244a09965c43e63355 Mon Sep 17 00:00:00 2001
From: Aaron Williams <aaron.williams@cavium.com>
Date: Mon, 8 Jan 2018 20:03:36 -0800
Subject: [PATCH 0731/1239] Cavium T81: SPI: Enabled support for 32-bit SPI
 flash

This is needed for the Interface Masters Sierra board.

Signed-off-by: Aaron Williams <aaron.williams@cavium.com>
---
 configs/t81_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/t81_defconfig b/configs/t81_defconfig
index 730fd362ed..70621bfaae 100644
--- a/configs/t81_defconfig
+++ b/configs/t81_defconfig
@@ -77,3 +77,4 @@ CONFIG_SYS_I2C_THUNDERX_SLAVE_ADDR=0x77
 CONFIG_ENV_VARS_UBOOT_CONFIG=y
 CONFIG_OF_LIBFDT=y
 CONFIG_OF_BOARD=y
+CONFIG_SPI_FLASH_4BAIS=y
-- 
2.29.0

